src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/annotation/suppresswarningsholder/Example3.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+  <module name="MemberName"/>
+  <module name="ConstantName"/>
+  <module name="ParameterNumber"/>
+  <module name="SuppressWarningsHolder">
+    <property name="aliasList" value=
+      "ParameterNumberCheck=paramnum"/>
```

Comment:
Please put `value=` on next line, to be on same line as value.

Please do this for example2 and example4